wait.sh

```diff
@@ -0,0 +1,6 @@
+until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306
```

Comment:
"Additionally, the hostname `nftbackend-mysql-1.nftbackend_default` doesn't appear to be the correct database hostname. The script functions properly after replacing the hostname with  'mysql'."
```suggestion
diff --git a/wait.sh b/wait.sh
index 03de611..14a7ee3 100644
--- a/wait.sh
+++ b/wait.sh
@@ -1,4 +1,4 @@
-until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306
+until nc -z -v -w30 "mysql" 3306
 do
   echo "Waiting a second until the database is receiving connections..."
   # wait for a second before checking again

```

Reply:
I did not clean my old network I guess